schema.graphql

```diff
@@ -171,11 +171,11 @@ type ProposedArtistAddressesAndSplit @entity {
   createdAt: BigInt!
 }
 
-type EngineRegistry @entity {
-  "Unique identifier made up of the Engine Registry's contract address"
+type CoreRegistry @entity {
+  "Unique identifier made up of the Core Registry's contract address (legacy name was Engine Registry). Note: For legacy MinterFilters, this is a dummy ID, equal to the address of the single core contract associated with the minter filter."
```

Comment:
Tiny nit: this would be more readable as a multi-line comment delimited by `"""`

Reply:
ah, thank you, good call!